include/igl/opengl/MeshGL.cpp

```diff
@@ -10,6 +10,11 @@
 #include "bind_vertex_attrib_array.h"
 #include "create_shader_program.h"
 #include "destroy_shader_program.h"
+#include "shaders/text.vert"
```

Comment:
Please include those directly into the source of `MeshGL::init`, as they are not really "headers".